atomic_reactor/schemas/config.json

```diff
@@ -651,12 +651,20 @@
             "description": "Do not report any media types other than these",
             "allOf": [{"$ref": "#/definitions/media_types"}]
           },
-          "blacklist_srpms": {
-              "description": "Don't include specified srpms to source container image, specify SRPM name",
-              "type": "array",
-              "items": {
-                "type": "string"
-              }
+          "denylist_srpms": {
+              "description": "Don't include srpms specified in denylist json with key",
+              "type": "object",
+              "properties": {
+                  "denylist_url": {
```

Comment:
Are we fine giving up possibility to specify extra denylisted RPMs directly in config map?

Reply:
yeah, no reason for us having separate list